tools/validate-xep0001-conformance.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+# Checks if provided file (filename to be provided as first argument) conforms to XEP-0001.
+# See https://github.com/xsf/xeps/issues/1235
+#
+# Expected to be executed from the directory that holds all xep XML files.
+# Requires one argument: the file name of the xep to be validated, eg:
+#
+# $ tools/validate-xep0001-conformance.sh xep-0010.xml
+#
+# exit status will be non-zero upon validation failure.
+#
+# requires: bash, xmllint
+
+set -e
+
+VALIDATION_RESULT=0
+
+# 1. Check DTD conformance against xep.dtd
+XEP_DTD="xep.dtd"
+if (xmllint --noout --dtdvalid "$XEP_DTD" "$1")
+then
+  echo "[PASS] DTD conformance against xep.dtd"
+else
+  echo "[FAIL] DTD conformance against xep.dtd"
+  VALIDATION_RESULT=1;
+fi
+
+FILE_NAME="$(basename -- $1)"
```

Comment:
Whenever you reference a variable (`$foobar`) and unless you know what you're doing, put it in double quotes.

But very good catch with `--` :-). Few people think of that (I often forget it, too).